### PR TITLE
New Curl Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Page: https://duck.co/ia/view/curl_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=curl cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/curl_cheat_sheet_mobile.png)

**Comment by mintsoft** [comment]
@vikashvverma can you add the following common used flags:
* `-s`
* `-S`
* `-k/--insecure`
* `--include` ?

Thanks

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=curl cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/curl_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=curl cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/curl_cheat_sheet_mobile.png)

**Comment by mintsoft** [comment]
ping @vikashvverma Just checking in, are you interested in finishing this one off?

**Comment by vikashvverma** [comment]
@mintsoft I am on tour right now. Will work on this after one or two week

File: share/goodie/cheat_sheets/json/curl.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    "metadata": {
+        "sourceName": "Cheatography",
+        "sourceUrl" : "http://www.cheatography.com/ankushagarwal11/cheat-sheets/curl-cheat-sheet/"
+    },
+
+    "aliases": [
+        "curl options"
+    ],
+
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/curl.json]: I feel like we should be using `terminal` here, seeing as these are entered on a command line

**Comment by moollaza** [comment]
@vikashvverma Thanks! This LGTM :+1:

@mintsoft care to make a task for your additions to be added?

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**